### ISSUE TITLE
Update upgrade images with pulp-admin packages and/or katello-agent

### ISSUE THREAD

**Issue by sghai** [SatelliteQE/satellite6-upgrade#186]
Recently, we have seen issues where users are facing issues when pulp-admin tools or katello-agent is installed on satellite server.

I see two use cases here:

a) katello-agent: In case of downstream satellite to pull contents from first connected satellite. Downstream satellite would be considered as content-host
b) pulp-admin to play with repos through pulp-admin tool

Reported issues: 
https://bugzilla.redhat.com/show_bug.cgi?id=1512959
https://bugzilla.redhat.com/show_bug.cgi?id=1594136

in short, make sure 'python-pulp-agent-lib' package is installed on satellite image. Lets make this change only on 6.3 image for now.
